methode-article-internal-components-mapper.yaml

```diff
@@ -194,4 +194,4 @@ validationConfiguration:
 
 appInfo:
     systemCode: "MethodeArticleInternalComponentsMapperApplication"
```

Comment:
Dewey has the system code `up-maicm`

Reply:
System code was updated, but somehow there was a conflict.
Anyways it should be all right now, please check.